cmd/influx/cli/cli.go

```diff
@@ -615,13 +615,13 @@ func (c *CommandLine) writeColumns(response *client.Response, w io.Writer) {
 		}
 
 		// Create a tabbed writer for each result as they won't always line up
-		w := new(tabwriter.Writer)
-		w.Init(os.Stdout, 0, 8, 1, '\t', 0)
+		writer := new(tabwriter.Writer)
```

Comment:
You can move the writer initialisation outside of the loops to the top of the function.